src/ORM/Query.php

```diff
@@ -242,6 +242,25 @@ public function select($fields = [], bool $overwrite = false)
         return parent::select($fields, $overwrite);
     }
 
+    /**
+     * Behaves the exact same as `select()` except adds the field to the list of fields selected and
+     * does not disable auto-selecting fields for Associations.
+     *
+     * Use this instead of calling `select()` then `enableAutoFields()` to re-enable auto-fields.
+     *
+     * @param \Cake\Database\ExpressionInterface|\Cake\ORM\Table|\Cake\ORM\Association|callable|array|string $fields Fields
+     * to be added to the list.
+     * @param bool $overwrite whether to reset fields with passed list or not
+     * @return $this
+     */
+    public function selectAlso($fields = [], bool $overwrite = false)
+    {
+        $this->select($fields, $overwrite);
+        $this->_autoFields = true;
```

Comment:
I tried to make this only enable if it wasn't previously disabled, but that ended up causing issues. I didn't want to introduce another complex flag.